src/conf_mode/service_dhcp-server.py

```diff
@@ -247,7 +248,11 @@ def verify(dhcp):
                         if mapping_config['ip_address'] in used_ips:
                             raise ConfigError(f'Configured IP address for static mapping "{mapping}" exists on another static mapping')
 
+                        if mapping_config['mac'] in used_mac:
+                            raise ConfigError(f'Configured MAC address for static mapping "{mapping}" exists on another static mapping')
+
                         used_ips.append(mapping_config['ip_address'])
+                        used_mac.append(mapping_config['mac'])
```

Comment:
This needs to be inside a check that the mac field is present. The code can reach this point with only a DUID and IP set.

Reply:
Good catch! We should have an identical if statement for the `mac` address key

Reply:
Ok i perform the corrections the IF statement should't be identical . acording the documentation you are able to set a mac address only and will assigned by the dynamic pool .

Reply:
I wonder if we should add a check for duplicate duids as well @sarthurdev?